trac-env/templates/custom_ticket.html

```diff
@@ -12,9 +12,9 @@ <h2>Please read this first:</h2>
       Report bugs and suggest documentation changes using this ticket tracker.
     </li>
     <li>
-      New feature proposals should first be discussed on the <a href="https://forum.djangoproject.com/c/internals/5">Django Forum</a>.
-      Once feedback has been gathered, you can use this ticket tracker to open a ticket for the new feature.
+      New feature proposals should be created on the <a href="https://github.com/orgs/django/projects/24/">new feature idea tracker</a>.
```

Comment:
```suggestion
      New feature proposals should be created on the <a href="https://github.com/django/new-features/issues/new/choose">new feature idea tracker</a>. The <a href="https://github.com/orgs/django/projects/24">project board view</a> can be used to view existing ideas and their progress..
```

Reply:
I think it's worth linking them to the place where the features are created or to the README. There's more guidance there on how to request a new feature. The kanban view is more for tracking progress and doesn't feel as helpful to people who want to request a new feature.

Reply:
One thing I was thinking with linking to the kanban view first is it might prompt people more to look for a similar issue. But I think your suggestion is probably better on the whole.

Reply:
That's fair. We do have the step to search for existing ideas in the new feature template text. :shrug: I'm good with whatever you decide here.